movo_common/si_utils/src/si_utils/head_jtas_test.py

```diff
@@ -112,7 +112,7 @@ def main():
     points = [list(current_angles), 0.0]
     for i in range(0,10):
         
-        pos = [random.uniform(-1.57,1.57),random.uniform(-1.57,1.57)]
+        pos = [0,random.uniform(-1.57,1.57)]
```

Comment:
why absolutely 0?

Reply:
Haven't realized I modified it, apologies